packages/react/src/field/docs/overview.mdx

```diff
@@ -100,4 +105,23 @@ function ExampleForm() {
 		</ul>
 	);
 }
+
+/**
+ * Example usage scrolling and focusing to a specific field on first mount
+ */
+function ExampleTwo() {
```

Comment:
We do not have any "living" examples currently where this hook would be beneficial, but I've added this snippet to the documentation site for now.